app/lib/git_repository.rb

```diff
@@ -409,8 +409,7 @@ def self.tmp_repo(connect_string)
   ####################################################################
 
   # Helper method to generate all the permissions for students for all groupings in all assignments.
-  # This is done as a single operation to mirror the SVN repo code. We found
-  # a substantial performance improvement by writing the auth file only once in the SVN case.
+  # This is done as a single operation to mirror the SVN repo code.
```

Comment:
I left it because it's a reminder of why we designed it this way in the first place. We wrote it this way in order to be as similar as possible to the way that subversion does things. I think the comment is still relevant but I can remove it if you'd prefer. We'll end up removing it when we stop writing the access file altogether anyway.

Reply:
Okay that's fine 👍